ecommerce/coupon_discounts/lib/coupon_discounts.rb

```diff
@@ -0,0 +1,15 @@
+require "infra"
+require_relative "coupon_discounts/commands/register_coupon"
+require_relative "coupon_discounts/events/coupon_registered"
+require_relative "coupon_discounts/services/on_coupon_register" #command handler
```

Comment:
we can live without this code comment - we try not to have them too many, as they propagate quickly as a habit

Reply:
I'll add something about it in README for future generations and remove the comments